README.md

```diff
@@ -16,7 +16,7 @@ Feel free to [create an issue](https://github.com/EddieJaoudeCommunity/EddieBot/
 
 - Timezone, listens for messages that contain 1:30pm UTC and replies with common timezones translation
 
-- Code of Conduct
+- Flags violations of the Code of Conduct.
```

Comment:
I think this refers to `^coc` which outputs the Code Of Conduct.
But good point, we should probably add- "Flags the use of non-inclusive language in the server" or something like that to the features